images/windows/templates/locals.windows.pkr.hcl

```diff
@@ -0,0 +1,12 @@
+locals {
+  image_properties    = var.image_sku == "2019-Datacenter" ? {
```

Comment:
instead of using `if` condition `a == b ? c : d == e ? f :g`, consider using maps in HCL.
It should be something like this:
```
image_properties_map = {
  "2019-Datacenter" = {
    image_os = "win19"
    os_disk_size_gb = var.os_disk_size_gb != 0 ? var.os_disk_size_gb : 256
  },
  "2022-Datacenter" = {
      image_os = "win22"Add commentMore actions
      os_disk_size_gb = var.os_disk_size_gb != 0 ? var.os_disk_size_gb : 256
  }
}
image_properties = image_properties_map[var.image_sku]
```